palo_alto_pan_os/help.md

```diff
@@ -641,6 +641,7 @@ When using the Add External Dynamic List action, a day and time must be chosen e
 
 # Version History
 
+* 1.5.7 - Default value of Commit action updated
```

Comment:
```suggestion
* 1.5.7 - Update to add the default value of Commit action
```

Reply:
it isn't actually adding the default value - it's fixing the default value which was incorrect.